packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -800,7 +800,9 @@ function ListView({
                             return (
                               <Td key={key}>
                                 <Typography textColor="neutral800">
-                                  {getDisplayName(rowData[name.split('.')[0]], formatMessage)}
+                                  {rowData[name.split('.')[0]]
```

Comment:
Could you add a comment explaining when user can empty? I suppose it is when an entity is created through the content API?